go/types/value_store_test.go

```diff
@@ -89,7 +100,7 @@ func TestCacheOnReadValue(t *testing.T) {
 	v := cvs2.ReadValue(r.TargetHash())
 	assert.True(bref.Equals(v))
 	assert.True(cvs2.isPresent(b.Hash()))
-	assert.True(cvs2.isPresent(b.Hash()))
+	assert.True(cvs2.isPresent(bref.Hash()))
```

Comment:
Nope, the test is verifying that, as a result of reading the Value `bref` out of `cvs2`, both `b` and `bref` are now in cvs2's cache of 'chunks I know about'